src/index.css

```diff
@@ -12,6 +12,17 @@ code {
     monospace;
 }
 
```

Comment:
Consider to move these styles to more specific css files so they are not global. Since this is very small project, this should be ok